plugins/tracker-resources/src/components/sprints/SprintBrowser.svelte

```diff
@@ -84,7 +84,10 @@
 
   async function update (viewlets: WithLookup<Viewlet>[], active: Ref<Viewlet> | null): Promise<void> {
     viewlet = viewlets.find((viewlet) => viewlet._id === active) ?? viewlets[0]
-    setActiveViewletId(viewlet._id)
+
+    if (viewlet) {
+      setActiveViewletId(viewlet._id)
```

Comment:
@BykhovDenis Should it be set to undefined in this case?

Reply:
I think we always should have least one viewlet

Reply:
`viewlets` are `[]` by default. So, we don't have viewlets until the viewlets request is completed.

Reply:
Should it be better to check for viewlets length before calling update in this case?

Reply:
Anyway, I think it's better to move this func to some util to avoid duplication throughout the project.